conan/cli/command.py

```diff
@@ -98,6 +98,15 @@ def _format(self, parser, info, *args):
 
         formatter(info)
 
+    def _raise_errors(self, info):
+        if info and type(info) == dict:
+            if "conan_error" in info and isinstance(info["conan_error"], Exception):
```

Comment:
The ``isinstance()`` might be just hiding user errors. If some code defines a ``conan_error``, but it is just a string, this will do nothing.